packages/node/Dockerfile

```diff
@@ -41,6 +40,7 @@ RUN mkdir -p .monitor && \
 
 # Make the user not ROOT
 USER 1000
+ENV TZ utc
```

Comment:
https://github.com/subquery/subql/actions/runs/10137262392/job/28027253016#step:10:882

 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 3)
 
 Not sure why still getting this from main ci, but we should update the format